test/hca/utils/validations.unit.spec.js

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai';
+
+import _ from 'lodash';
+
+import {
+  isValidSpouseInformation
```

Comment:
Nit: probably can put this whole thing on a single line since we're only importing one function?